servers/xr_server.cpp

```diff
@@ -352,6 +362,52 @@ PackedStringArray XRServer::get_suggested_pose_names(const StringName &p_tracker
 	return arr;
 }
 
+void XRServer::add_face_tracker(const StringName &p_path, Ref<XRFaceTracker> p_face_tracker) {
+	ERR_FAIL_COND(p_face_tracker.is_null());
+
+	if (!face_trackers.has(p_path)) {
+		// We don't have a tracker with this name, we're going to add it.
+		face_trackers[p_path] = p_face_tracker;
+		emit_signal(SNAME("face_tracker_added"), p_path, p_face_tracker);
+	} else if (face_trackers[p_path] != p_face_tracker) {
+		// We already have a tracker with this name, we're going to replace it.
+		face_trackers[p_path] = p_face_tracker;
+		emit_signal(SNAME("face_tracker_updated"), p_path, p_face_tracker);
+	}
+}
+
+void XRServer::remove_face_tracker(const StringName &p_path) {
+	// Skip if no provider is found
+	if (!face_trackers.has(p_path)) {
+		return;
+	}
+
+	// we send the signal right before removing it
+	emit_signal(SNAME("face_tracker_removed"), p_path);
+
+	// and remove it
+	face_trackers.erase(p_path);
+}
+
+PackedStringArray XRServer::get_face_tracker_paths() const {
+	PackedStringArray arr;
+
+	for (int i = 0; i < face_trackers.size(); i++) {
+		arr.push_back(face_trackers.get_key_at_index(i));
+	}
+
+	return arr;
+}
+
+Ref<XRFaceTracker> XRServer::get_face_tracker(const StringName &p_path) const {
+	// Skip if no provider is found
```

Comment:
```suggestion
	// Skip if no provider is found.
```

Reply:
Done - also removed the older "provider" terminology.